packages/Chips/src/index.tsx

```diff
@@ -0,0 +1,62 @@
+import withUtils from "@blaze-react/utils";
+import React, { Fragment, ReactElement, useState } from "react";
+import ChipAvatar from "./ChipAvatar";
+import ChipIcon from "./ChipIcon";
+import ChipLabel from "./ChipLabel";
+
+interface IChipsProps {
+  children: JSX.Element | JSX.Element[];
+  modifiers: string[];
```

Comment:
modifiers should be optional if you have a default props, should be like ```modifiers?```or if should be always necessary then delete default props